db/db_test.cc

```diff
@@ -70,9 +70,11 @@
 
 namespace ROCKSDB_NAMESPACE {
 
+// Note that this hope test disables fsync for files and directories.
```

Comment:
"whole test" I assume.

And maybe say "for speed" though that is in the callee comment